WooCommerce/WooCommerceTests/Authentication/NotWPAccountViewModelTests.swift

```diff
@@ -73,17 +105,30 @@ final class NotWPAccountViewModelTests: XCTestCase {
         XCTAssertEqual(primaryButtonTitle, Expectations.restartLoginTitle)
     }
 
-    func test_primary_button_title_is_login_with_site_address_for_invalidWPComEmail_from_wpCom_error() {
+    func test_primary_button_title_is_login_with_site_address_for_invalidWPComEmail_from_wpCom_error_when_simplified_login_feature_flag_is_off() {
         // Given
-        let viewModel = NotWPAccountViewModel(error: SignInError.invalidWPComEmail(source: .wpCom))
-
+        let featureFlagService = MockFeatureFlagService(isSimplifiedLoginFlowI1Enabled: false)
+        let viewModel = NotWPAccountViewModel(error: SignInError.invalidWPComEmail(source: .wpCom),
+                                              featureFlagService: featureFlagService)
         // When
         let primaryButtonTitle = viewModel.primaryButtonTitle
 
         // Then
         XCTAssertEqual(primaryButtonTitle, Expectations.loginWithSiteAddressTitle)
     }
 
+    func test_primary_button_title_is_login_with_site_address_for_invalidWPComEmail_from_wpCom_error_when_simplified_login_feature_flag_is_on() {
```

Comment:
Nit: should the test case expect "Create an Account" instead?

```suggestion
    func test_primary_button_title_is_create_an_account_when_simplified_login_feature_flag_is_on() {
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/7954/commits/525f03c0c4c38f2a323cbf1d1db5c9b989119206